icu4c/source/i18n/tibetancal.h

```diff
@@ -0,0 +1,313 @@
+// © 2016 and later: Unicode, Inc. and others.
+// License & terms of use: http://www.unicode.org/copyright.html
+/*
+ *****************************************************************************
+ * Copyright (C) 2003-2021, International Business Machines Corporation
+ * and others. All Rights Reserved.
+ *****************************************************************************
+ *
+ * File TIBETANCAL.H
+ *****************************************************************************
+ */
+
+#ifndef TIBETANCAL_H
+#define TIBETANCAL_H
+
+#include "unicode/utypes.h"
+
+#if !UCONFIG_NO_FORMATTING
+
+#include "unicode/calendar.h"
+
+U_NAMESPACE_BEGIN
+
+/**
+*  Concrete class which provides the Tibetan calendar.
+*
+*
+*
+*
+*
+*
+*
+*/
+
+
+class U_I18N_API TibetanCalendar : public Calendar {
+public:
+  /**
+   * Useful constants for TibetanCalendar.
+   * @internal
+   */
+
+
+  /**
+   * Calendar type - phugpa or tsurphu
+   * @internal 
+   */
+  enum ECalculationType {
```

Comment:
please remove this enum. I clean up the rest of Calendar subclass in https://github.com/unicode-org/icu/pull/2215 please keep that. thanks